build.gradle

```diff
@@ -37,6 +37,7 @@ buildscript {
         roomVersion = '1.1.1'
         runnerVersion = '1.0.1'
         supportLibraryVersion = '28.0.0-rc01'
+        uiautomator = "2.1.3"
```

Comment:
Thanks for moving this! Can you also add a `Version` suffix and change capitalization so it reads `uiAutomatorVersion`?

Reply:
👍